From 16ebe1f5232621d8894aa6c6210fdf2fc9b54a84 Mon Sep 17 00:00:00 2001
From: Brendan O'Dea <bod@debian.org>
Date: Fri, 18 Mar 2005 22:22:25 +1100
Subject: Replace generic man(1) instructions with Debian-specific
 information.

Indicate that the user needs to install the perl-doc package.

Patch-Name: debian/doc_info.diff
---
 pod/perl.pod |   12 ++++++++++--
 1 files changed, 10 insertions(+), 2 deletions(-)

diff --git a/pod/perl.pod b/pod/perl.pod
index 29cabf1..529ad6f 100644
--- a/pod/perl.pod
+++ b/pod/perl.pod
@@ -261,8 +261,16 @@ For ease of access, the Perl manual has been split up into several sections.
     perlwin32		Perl notes for Windows
 
 
-On a Unix-like system, these documentation files will usually also be
-available as manpages for use with the F<man> program.
+On Debian systems, you need to install the B<perl-doc> package which
+contains the majority of the standard Perl documentation and the
+F<perldoc> program.
+
+Extensive additional documentation for Perl modules is available, both
+those distributed with Perl and third-party modules which are packaged
+or locally installed.
+
+You should be able to view Perl's documentation with your man(1)
+program or perldoc(1).
 
 In general, if something strange has gone wrong with your program and you're
 not sure where you should look for help, try the B<-w> switch first.  It will
